website/api/models/VantaConnection.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * VantaConnection.js
+ *
+ * @description :: An organization who is a customer of Vanta.
+ * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
+ */
+
+module.exports = {
+
+  attributes: {
+
+    //  ╔═╗╦═╗╦╔╦╗╦╔╦╗╦╦  ╦╔═╗╔═╗
+    //  ╠═╝╠╦╝║║║║║ ║ ║╚╗╔╝║╣ ╚═╗
+    //  ╩  ╩╚═╩╩ ╩╩ ╩ ╩ ╚╝ ╚═╝╚═╝
+    emailAddress: {
+      description: 'The email address provided when this Vanta connection was created.',
```

Comment:
Add extended description: Why is this being captureD?  (In case we need to contact the user about any issues that come up)